proto/vtctldata.proto

```diff
@@ -374,6 +374,11 @@ message ApplyVSchemaRequest {
 
 message ApplyVSchemaResponse {
   vschema.Keyspace v_schema = 1;
+  map<string, ParamList> unknown_vindex_params = 2;
```

Comment:
Can you please add a comment to this field (`UnknownVindexParams is ...`) and can you please explain the format here? I'm not sure I understand what in this `map[string]([]string)`.

Reply:
Added a comment in 5e4fa13267cb

In terms of what goes in here. The VSchema allows you defined `params` on Vindexes in the form of `map[string]string`. This makes it very easy for users to put in parameters that aren't recognized, e.g. due to typos.  Until relatively recently, Vitess would happily accept unknown vindex params, and not report it as an issue. Vitess now has the internal ability to track and report unknown params. However, `vtctld` has not yet caught up.

This PR addresses that by adding a `map[string, ParamList]` field to the `ApplyVSchema` gRPC response. This contents will map the name(s) of user-defined vindex(es) to the names of any unknown params found in the `params` block of those vindexes. The somewhat awkward way of defining a "map to a list of strings" here is due to limitations in protobuf.